src/monitor.c

```diff
@@ -458,3 +470,59 @@ struct wlr_box monitor_get_active_geom(struct monitor *m)
     struct wlr_box geom = tag_get_active_geom(tag);
     return geom;
 }
+
+void handle_output_mgr_apply(struct wl_listener *listener, void *data)
+{
+	struct wlr_output_configuration_v1 *config = data;
+	handle_output_mgr_apply_test(config, false);
+}
+
+// apply_output_config
+void handle_output_mgr_apply_test(struct wlr_output_configuration_v1 *config, bool test)
+{
+	struct wlr_output_configuration_head_v1 *config_head;
+	bool ok = true;
+
+	wl_list_for_each(config_head, &config->heads, link) {
+		struct wlr_output *wlr_output = config_head->state.output;
+
+		wlr_output_enable(wlr_output, config_head->state.enabled);
+
+		if (config_head->state.enabled) {
```

Comment:
I don't know but instead of using multiple if statements after eachother you should rather use elseifs

Reply:
dont think i can here, if i'm not mistaken all if statements need to be done. If I made them elseifs wouldn't the later one not be done if the first one gets done?

Reply:
Oh sorry then you are right